code-name-taran/src/com/spaghettisoft/component/hangman/WordToGuess.java

```diff
@@ -0,0 +1,109 @@
+package com.spaghettisoft.component.hangman;
+
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.File;
+
+public class WordToGuess {

```

Comment:
The abstraction of the WordToGuess class should be made clearer. You should separate the logic of loading random word from a file in different class. The WordToGuess class should accept string representing the word in the custructor. The class should have the methods like "void guessCharacter(char c)", "String getMaskedRepresentation()" and "boolean isGuessed()". This methods should be used inside the HangmanGame class.